ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -281,6 +361,20 @@ export default class TransactionListItemDetails extends PureComponent {
                 primaryCurrency={primaryCurrency}
                 className="transaction-list-item-details__transaction-breakdown"
               />
+              {
+                ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+                transactionNote && transactionNote.length !== 0 && (
+                  <div className="transaction-list-item-details__transaction-breakdown">
+                    <h4 className="transaction-breakdown__title">
+                      {t('transactionNote')}
+                    </h4>
+                    <p className="transaction-breakdown__description">
+                      {transactionNote}
+                    </p>
```

Comment:
Suggestion: we can use Text component to take advantage of some build-in styles

Reply:
Fixed